omnibus/config/software/cacerts.rb

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2012-2014 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "cacerts"
+
+# We always pull the latest version,
+# so the hashsum check will break every time the file is updated on the remote
+default_version "latest"
+
+source url: "https://curl.haxx.se/ca/cacert.pem",
+       sha256: "cb2eca3fbfa232c9e3874e3852d43b33589f27face98eef10242a853d83a437a"
+
```

Comment:
Isn't this file already present in `omnibus-software` [here](https://github.com/DataDog/omnibus-software/blob/master/config/software/cacerts.rb)? I fail to see any difference except for the `target_filename` that was removed. Is there a specific reason we need this definition in the `datadog-agent` repository?

Reply:
We should use the version of the file in `omnibus-software` (i.e. with `target_filename`) otherwise the `build` step will not copy the file as expected with the `omnibus-ruby` version that we use on `master`.

If there's a good reason to keep a separate software def in this repo, let's update this file accordingly. Otherwise let's simply rely on the software def of `omnibus-software`.

Reply:
yeah, let me remove that. Added by accident to get around the fact that `omnibus-software` was broken as I was developing.